### ISSUE TITLE
Interface width/height preferences keep going back to default

### ISSUE THREAD

**Issue by jtaseff** [ClaudeMetz/FactoryPlanner#371]
### Problem Description

I keep opening Factory Planner, and changing the interface size to show more rows and products.
After a short time, seems to be a few minutes, when I open it the view size has gone back to the default smallest numbers.

### Reproduction

Haven't been able to reproduce exactly what causes it, doesn't seem to be autosave or pausing or alt-tab out of the game.... but has happened at least 5 times in an hour on a new Space Age run

**Comment by jtaseff** [comment]
Edited: found how to reproduce it.

**Comment by ClaudeMetz** [comment]
Yeah this is an issue I know about, you're probably on Linux right? The mod tries to keep its window within a reasonable size so it always fits. What your OS does when you alt-tab is resize the Factorio window to be very small, which triggers this resizing logic.

I tried fixing this before by only running the resizing when you start a save for the first time, but that wasn't as easy as I thought. I'll keep this card around and will give it a second attempt at some point.

Appreciate the report!

**Comment by jtaseff** [comment]
Windows 10 actually. 
If it helps anything, display is 4K, Windows UI scaling is set to 250%, factorio UI scaling is manual (pixels) 200% if that's messing with things.
Cheers, thanks for all the hard work getting it running for 2.0!

**Comment by ClaudeMetz** [comment]
Ah interesting, it's probably a similar issue though. We'll see!
